tasks/init

```diff
@@ -33,29 +35,37 @@ docker network create --driver bridge ${vp_network_name} 2>/dev/null || echo ""
 # Spin up consul to back vault and then spin up vault.
 docker run -d -p ${VP_CONSUL_PORT}:8500 --network ${vp_network_name} -e CONSUL_BIND_INTERFACE=eth0 --name vp-consul1 consul && docker run -d -p ${VP_VAULT_PORT}:8200 --cap-add=IPC_LOCK --network ${vp_network_name} --name vp-vault1 -e VAULT_ADDR=http://127.0.0.1:8200 -e 'VAULT_LOCAL_CONFIG={"backend": {"consul": {"path": "/vault/", "check_timeout": "5s", "max_parallel": "128", "address": "http://vp-consul1:8500", "cluster_addr": "http://vp-vault1:8200", "api_addr": "http://vp-vault1:8200"}}, "listener": { "tcp": { "address":"0.0.0.0:8200", "tls_disable":1}}, "default_lease_ttl": "168h", "max_lease_ttl": "720h"}' vault server
 
```

Comment:
would it make sense to have a block at the top of this script that verifies that the required commands are installed correctly? e.g. docker, vault, curl, jq

Reply:
yes great idea! I was thinking about adding those assumed dependencies in the README too

Reply:
Updated with this. Note that I'm only using vault inside the vault docker image so it's not a dep, only docker, curl, and jq are.